src/App/App.tsx

```diff
@@ -56,94 +62,134 @@ class App extends Component {
         <Row id="main-page-row">
           <Col>
             <Switch>
+              {/* Home Page view */}
               <ConnectedPrivateRoute
                 disableLoginProtection={DISABLE_LOGIN_PROTECTION}
                 exact={true}
                 path="/"
                 component={Home}
               />
+
+              {/* Active IRS Plans list view */}
               <ConnectedPrivateRoute
                 disableLoginProtection={DISABLE_LOGIN_PROTECTION}
                 exact={true}
                 path={INTERVENTION_IRS_URL}
                 component={IrsPlans}
               />
 
+              {/* Draft IRS Plans list view */}
+              <ConnectedPrivateRoute
+                disableLoginProtection={DISABLE_LOGIN_PROTECTION}
+                exact={true}
+                path={INTERVENTION_IRS_DRAFTS_URL}
+                component={IrsPlans}
+              />
+
+              {/* New IRS Plan form view */}
               <ConnectedPrivateRoute
                 disableLoginProtection={DISABLE_LOGIN_PROTECTION}
                 exact={true}
                 path={NEW_IRS_PLAN_URL}
                 component={NewIRSPlan}
               />
 
+              {/* Draft IRS Plan Jurisdiction Selection view */}
+              <ConnectedPrivateRoute
+                disableLoginProtection={DISABLE_LOGIN_PROTECTION}
+                exact={true}
+                path={`${DRAFT_IRS_PLAN_URL}/:id`}
+                component={IrsPlan}
+              />
+
+              {/* Draft IRS Plan Team Assignment view */}
               <ConnectedPrivateRoute
                 disableLoginProtection={DISABLE_LOGIN_PROTECTION}
                 exact={true}
-                path={`${INTERVENTION_IRS_URL}/draft/:id`}
+                path={`${ACTIVE_IRS_PLAN_URL}/:id`}
                 component={IrsPlan}
               />
 
+              {/* IRS Reporting plan table view */}
+              <ConnectedPrivateRoute
+                disableLoginProtection={DISABLE_LOGIN_PROTECTION}
+                exact={true}
+                path={`${REPORT_IRS_PLAN_URL}/:id`}
+                component={IrsReport}
+              />
+
+              {/* IRS Reporting Map view */}
               <ConnectedPrivateRoute
                 disableLoginProtection={DISABLE_LOGIN_PROTECTION}
                 exact={true}
-                path={`${INTERVENTION_IRS_URL}/plan/:id`}
+                path={`${ACTIVE_IRS_PLAN_URL}/:id/${MAP}/:jurisdictionId`}
                 component={IrsPlan}
```

Comment:
Are we planning to reuse IrsPlan for IRS Reporting?

Reply:
@moshthepitt Nope, IRS Reporting Table page has its own container. If we open an `active` IRS plan in the IrsPlan module it should render a table for assigning teams to plan-jurisdictions. (once we build it)